typespec-tests/src/test/java/com/azure/resourcemanager/models/resources/ArmResourceTest.java

```diff
@@ -93,17 +91,17 @@ public void testTopLevelTrackedResource() {
         Assertions.assertEquals(RESOURCE_PROVISIONING_STATE, topLevelTrackedResource.properties().provisioningState());
 
         // TopLevelTrackedResources.update
-        topLevelTrackedResource.update()
-                .withProperties(
-                        new TopLevelTrackedResourceUpdateProperties().withDescription(RESOURCE_DESCRIPTION_VALID2))
-                .apply();
-        Assertions.assertEquals(TOP_LEVEL_TRACKED_RESOURCE_ID, topLevelTrackedResource.id());
-        Assertions.assertEquals(TOP_LEVEL_TRACKED_RESOURCE_NAME, topLevelTrackedResource.name());
-        Assertions.assertEquals(TOP_LEVEL_TRACKED_RESOURCE_TYPE, topLevelTrackedResource.type());
-        Assertions.assertEquals(TOP_LEVEL_TRACKED_RESOURCE_REGION, topLevelTrackedResource.region());
-        Assertions.assertNotNull(topLevelTrackedResource.properties());
-        Assertions.assertEquals(RESOURCE_DESCRIPTION_VALID2, topLevelTrackedResource.properties().description());
-        Assertions.assertEquals(RESOURCE_PROVISIONING_STATE, topLevelTrackedResource.properties().provisioningState());
+//        topLevelTrackedResource.update()
+//                .withProperties(
+//                        new TopLevelTrackedResourceProperties().withDescription(RESOURCE_DESCRIPTION_VALID2))
+//                .apply();
+//        Assertions.assertEquals(TOP_LEVEL_TRACKED_RESOURCE_ID, topLevelTrackedResource.id());
+//        Assertions.assertEquals(TOP_LEVEL_TRACKED_RESOURCE_NAME, topLevelTrackedResource.name());
+//        Assertions.assertEquals(TOP_LEVEL_TRACKED_RESOURCE_TYPE, topLevelTrackedResource.type());
+//        Assertions.assertEquals(TOP_LEVEL_TRACKED_RESOURCE_REGION, topLevelTrackedResource.region());
+//        Assertions.assertNotNull(topLevelTrackedResource.properties());
+//        Assertions.assertEquals(RESOURCE_DESCRIPTION_VALID2, topLevelTrackedResource.properties().description());
+//        Assertions.assertEquals(RESOURCE_PROVISIONING_STATE, topLevelTrackedResource.properties().provisioningState());
```

Comment:
I've temporarily disabled this call.

@XiaofeiCao you may need to update the verify logic in cadl-ranch.

```
com.azure.core.management.exception.ManagementException: Status code 400, "{"message":"Body provided doesn't match expected body","expected":{"properties":{"description":"valid2"}},"actual":{"location":"eastus","properties":{"description":"valid2"}}}"
```

Reply:
PR: https://github.com/Azure/cadl-ranch/pull/638